specification/dataprotection/resource-manager/Microsoft.DataProtection/preview/2022-03-31-preview/examples/BackupInstanceOperations/ListBackupInstancesExtensionRouting.json

```diff
@@ -0,0 +1,53 @@
+{
+  "parameters": {
+    "subscriptionId": "04cf684a-d41f-4550-9f70-7708a3a2283b",
+    "resourceGroupName": "000pikumar",
+    "resourceProviderNamespace": "Microsoft.DBforPostgreSQL",
+    "resourceType": "servers",
+    "resourceName": "viveksipgtest",
+    "api-version": "2022-03-31-preview"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "/subscriptions/04cf684a-d41f-4550-9f70-7708a3a2283b/resourceGroups/000pikumar/providers/Microsoft.DataProtection/backupVaults/PratikPrivatePreviewVault1/backupInstances/harshitbi2",
```

Comment:
>/subscriptions/04cf684a-d41f-4550-9f70-7708a3a2283b/resourceGroups/000pikumar/providers/Microsoft.DataProtection/backupVaults/PratikPrivatePreviewVault1/backupInstances/harshitbi2 [](http://example.com/codeflow?start=19&length=179)

The resource ID is invalid. The backupInstances resource should be an extension of the request scope (the sql server).
It should be
/subscriptions/04cf684a-d41f-4550-9f70-7708a3a2283b/resourceGroups/000pikumar/providers/Microsoft.DBforPostgreSQL/servers/viveksipgtest/providers/Microsoft.DataProtection/backupVaults/PratikPrivatePreviewVault1/backupInstances/harshitbi2

Reply:
thanks @j5lim for pointing it out, I've fixed the value of id in the example now